app/src/main/java/com/example/mpweather/MainActivity.kt

```diff
@@ -117,4 +115,9 @@ class MainActivity : AppCompatActivity() {
             requestQueue.add(stringRequest)
         }
     }
+    //An object that holds the variable for the number that is used to get the tempreture in °C
+    companion object{
```

Comment:
I created a companion object which holds the constant, you cannot just put the constant in the mainActivity, it has to be in an object or on top level.